src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE books (
+    id int NOT NULL AUTO_INCREMENT,
+    title varchar(255) NOT NULL,
+    price double,
+    PRIMARY KEY (ID)
+);
```

Comment:
add blank line to avoid red sign at the github
```suggestion
);

```